oak_functions/gcp/weather_lookup_api_config.yaml

```diff
@@ -0,0 +1,28 @@
+type: google.api.Service
+config_version: 3
+
+#
+# Name of the service configuration.
+#
+name: '*.apigateway.endpoints.oak-ci.cloud.goog'
+
+#
+# API title to appear in the user interface (Google Cloud Console).
+#
+title: Oak Functions gRPC API Gateway
+apis:
+  - name: oak.functions.server.GrpcHandler
```

Comment:
Is this just the protobuf package in the gRPC definition?

Reply:
It is the fully qualified service name for the service in the proto file, so the package name and service name concatenated.